doc/rst/usingchapel/chplenv.rst

```diff
@@ -655,7 +655,14 @@ CHPL_UNWIND
        ========= =======================================================
 
    If unset, ``CHPL_UNWIND`` defaults to ``none``
-
+ 
+CHPL_RT_UNWIND 
```

Comment:
Drop this header, so that this is just part of the `CHPL_UNWIND` documentation. In fact, you could throw this addition into a note, using `.. note::` (grep for that string for other examples in the docs)